app/views/fee_items/_fee_item_table.html.erb

```diff
@@ -1,21 +1,30 @@
 <table class="govuk-table">
   <thead class="govuk-table__head">
     <tr class="govuk-table__row">
-      <th scope="col" class="govuk-table__header">Item</th>
-      <th scope="col" class="govuk-table__header">Detail</th>
+      <th scope="col" class="govuk-table__header"><%= t(".item") %></th>
+      <th scope="col" class="govuk-table__header"><%= t(".detail") %></th>
     </tr>
   </thead>
   <tbody class="govuk-table__body">
     <tr class="govuk-table__row">
-      <th scope="row" class="govuk-table__header">Fee</th>
-      <td class="govuk-table__cell"><%= number_to_currency (@planning_application.payment_amount), unit: '£' %></td>
+      <th scope="row" class="govuk-table__header"><%= t(".fee_paid") %></th>
```

Comment:
This is the only real change in this file - "Fee" -> "Fee paid". But I moved everything into locale files while I was at it.